src/Pages/Dashboard/Main/Main.tsx

```diff
@@ -43,10 +50,10 @@ export const Main = ({ selectedChatRoomId }: Props) => {
         <button
           type="submit"
           onClick={handleClick}
-          disabled={loading}
+          disabled={status === "loading"}
```

Comment:
if we change the status name from `loading` to `LOADING` in useMutation, we will have to go to all the occurrences and change `loading` to `LOADING`. Therefore, we should use constant